app/components/WidgetDynamicContent/WidgetDynamicContent.tsx

```diff
@@ -239,13 +241,17 @@ export default class WidgetDynamicContent extends Component {
 
     renderReact() {
         const { data, widget } = this.props;
+        const showErrorWithPopup = !widget.definition.showBorder;
```

Comment:
Wouldn't it be more accurate to say:
```suggestion
        const showErrorInPopup = !widget.definition.showBorder;
```
?

Reply:
Hmm... Wait, didn't we agree to show all errors in popup when we have borderless widgets and inside widget box just show "<widget-id> widget"?

Reply:
Oh yeah, you're right 😁 
In that case it's 100% valid 🎉 

I'll rename it here and in the `WidgetErrorMessage` component aswell 💪

Reply:
It has been renamed 🎉